website/project/views/node.py

```diff
@@ -781,6 +784,7 @@ def _view_project(node, auth, primary=False):
             },
             'alternative_citations': [citation.to_json() for citation in node.alternative_citations],
             'has_draft_registrations': node.has_active_draft_registrations,
+            'contributors': contributor_ids
```

Comment:
I am pretty sure its used in another PR (removing contributors) that used to be a part of this one.  I will pull it.